tests/test_vmec.py

```diff
@@ -371,7 +370,7 @@ def test_vmec_save_asym(TmpDir):
     VMECIO.save(eq, output_path)
 
 
-@pytest.mark.unit
+@pytest.mark.regression
```

Comment:
was this just because it takes a while to run?

Reply:
It consistently got cancelled (due to out of memory) on github when it was marked as a unit test. Thought shifting it to regression might allocate memory to the tests better